quality_control_mrp_oca/views/qc_inspection_view.xml

```diff
@@ -5,7 +5,7 @@
     <record model="ir.ui.view" id="qc_inspection_form_view_production">
         <field name="name">qc.inspection.form.view.production</field>
         <field name="model">qc.inspection</field>
-        <field name="inherit_id" ref="quality_control.qc_inspection_form_view" />
+        <field name="inherit_id" ref="quality_control_oca.qc_inspection_form_view" />
```

Comment:
I think you can remove the module part, as relative links may work here (the same for the rest)